File: Python3/simple_program.py
# This is python simaple program

# For this program we not required any imports

####NOTE#####
#   1. Since python is interpreted language so their is no entry function it runs from starting line
#   2. Indentation is very important in python.  


def greeting(name: str):
    print(f"Hi, {name}")
    print("Congrats you have successfully run the python program")


if __name__ == "__main__":  # Here I check is this file main run file
    print("Welcome to the Python World")       # display a message on console

    name = input("Enter your name: ")        # accepting string value from user inputs
    greeting(name)                           # Calling user defined function

File: GoLang/simpleProgram.go
// Go lang simple program which greet the user

// Every go file must specify it package
package main // If specify package main Go compiler create exe file

// import required package
import "fmt" // formating package

/**
Syntax

func FunctionName( Parameter ) Returntype{}
*/
func greeting(name string) {
	fmt.Println("Hi, " + name)
	fmt.Println("Congrats you have successfully run the Go program")
}

// Go entry point function
func main() {
	fmt.Println("Welcome to Golang World")

	var name string // declaring variable
	fmt.Print("Enter your name: ")
	fmt.Scanln(&name) // accepting user input

	greeting(name)
}
